doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -386,34 +360,52 @@ on the shape. In particular, the following geometries can be defined so far:</p>
 </ul>
 
-</td></tr>
-<tr><td style="padding-right: 40px; width: 140px;">coords</td><td>
-
-<p>List of coordinates (separated by <code>;</code>) are interpreted depending on the geometry parameter as follows:</p>
-
 <ul>
   <li>In the case of <code>circle</code>geometry, the parameter includes just one element, which
   is the center of the circle.</li>
   <li>In the case of <code>polygon</code>includes a list of elements, which are the vertices of
   the polygon.</li>
 </ul>
+</p>
 
-</td></tr>
-<tr><td style="padding-right: 40px; width: 140px;">attrs</td>
-    <td>Comma-separated list of attribute names which data will be included in the response. If this parameter is not
-    included, all the attributes are retrieved.</td>
-</tr>
-<tr><td style="padding-right: 40px; width: 140px;">limit</td><td>Limit the number of entities to be retrieved</td></tr>
-<tr><td style="padding-right: 40px; width: 140px;">offset</td><td>Skip a number of records</td></tr>
-<tr><td style="padding-right: 40px; width: 140px;">options</td><td>
-This operation supports two options:
+# REST API
 
-<ul>
-  <li><code>count</code>: when used, the total number of entities is returned in the response as a HTTP header named <code>X-Total-Count</code>.</li>
-  <li><code>canonical</code>: when used, the response payload uses the "canonical form". See JSON entity representation format section
-  for details.</li>
-</td></tr>
-</table>
+# Group NGSIv2
+
+## API Entry Point [/v2]
+
+### Retrieve Entry Points [GET /v2]
+
+This resource does not have any attributes. Instead it offers the initial
+API affordances in the form of the links in the JSON body.
+
+It is recommended to follow the “url” link values,
+[Link](https://tools.ietf.org/html/rfc5988) or Location headers where
+applicable to retrieve resources. Instead of constructing your own URLs,
+to keep your client decoupled from implementation details.
+
++ Attributes(object)
+    + entities_url (required, string) - URL which points to the entities resource
+    + types_url (required, string) - URL which points to the types resource
+    + subscriptions_url (required, string) - URL which points to the subscriptions resource
+    + registrations_url (required, string) - URL which points to the registrations resource
+
++ Response 200 (application/json)
+
+        {
+            "entities_url":      "/v2/entities",
+            "types_url":         "/v2/types",
+            "subscriptions_url": "/v2/subscriptions",
+            "registrations_url": "/v2/registrations"
+        }
+
+## Entities [/v2/entities{?limit,offset,options,type,id,idPattern,q,geometry,coords,attrs}]
+
+### List entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,q,geometry,coords,attrs}]
+
+Retrieves a list of entities which match criteria defined by the following parameters: `id`, `type`, `idPattern`, `q`,
+`geometry` and `coords` attribute (see below for a detailed description of these parameters). A given entity
```

Comment:
I think the "see below for a detailed description of these parameters" is now obsolete... the "link" should point to somewhere (part in the Simple Query Language description).